pl-faded-parsons.js

```diff
@@ -523,6 +523,34 @@ class ParsonsWidget {
           },
         }),
       );
+    
+    // add uids to each line and blank for logging ////////////////////////////
+    //  + add logging hooks for blank edits       ////////////////////////////
+    $(widget.config.main)
+      .find(".codeline-tray")
+      .each((trayNumber, tray) =>
+        $(tray)
+          .find("li.codeline")
+          .each((codelineNumber, codeline) => {
+            if ($(codeline).attr("logging-id")) return;
```

Comment:
I know I wrote this line but I don't know if it makes sense in your version of logging. I'm not sure if the system remembers ids between submissions, so I don't know if this return condition is ever met...

Reply:
Made it so that logs are preserved between submissions, tested it on a dummy course with an FPP